src/domain/eventSearch/__tests__/EventSearchPageContainer.test.tsx

```diff
@@ -263,4 +264,4 @@ it('should scroll to result list on mobile screen', async () => {
   expect(scroller.scrollTo).toBeCalled();
 });
 
-it('searches events correctly with event place in path, e.g. /fi/annantalo', () => {});
+//it('searches events correctly with event place in path, e.g. /fi/annantalo', () => {});
```

Comment:
hmm seems like I forgot this here earlier :D

Reply:
i didnt remove, so if needed, it can be implemented later, but still present in the code

Reply:
👍